packages/core-web/src/index.js

```diff
@@ -52,38 +52,75 @@ function detectAndApplyFixedHeaderStyles() {
       }`);
   insertCss(`.nav-menu-open { max-height: calc(100% - ${headerHeight}px); }`);
 
+  const resizeHeader = () => {
```

Comment:
```suggestion
  const adjustHeaderClasses = () => {
```

since the header itself isn't resized =P